packages/nimble-components/src/rich-text-mention/base/models/mention-internals.ts

```diff
@@ -0,0 +1,36 @@
+import { observable } from '@microsoft/fast-element';
+
+export interface MentionInternalsOptions {
+    readonly icon: string;
+    readonly character: string;
+    readonly pattern: string;
+}
+
+/**
+ * Internal mention state
+ */
+export class MentionInternals<TMentionConfig> {
+    /**
+     * The relevant, static configuration which will hold mentioning info of view, character, icon and pattern
+     */
```

Comment:
this doc comment is on mentionConfig which is not the static configuration

Reply:
Updated